doc/tractography_methods_list.rst

```diff
@@ -0,0 +1,28 @@
+.. list-table:: Fiber tracking methods available in DIPY.
+   :widths: 10 6 12 10
+   :header-rows: 1
+
+   * - Method
+     - Local vs. Global
+     - Deterministic vs. Probabilistic
+     - References
+   * - :ref:`Local deterministic <sphx_glr_examples_built_fiber_tracking_tracking_deterministic.py>` (EuDX)
+     - Local
+     - Deterministic
+     - :cite:t:`Garyfallidis2012b`
+   * - :ref:`Local probabilistic <sphx_glr_examples_built_fiber_tracking_tracking_probabilistic.py>`
+     - Local
+     - Probabilistic
+     - :cite:t:`Behrens2003`, :cite:t:`Behrens2007`
+   * - :ref:`Bootstrap tracking <sphx_glr_examples_built_fiber_tracking_tracking_bootstrap_peaks.py>`
+     - Local
+     - Probabilistic
+     - :cite:t:`Berman2008`
+   * - :ref:`PFT <sphx_glr_examples_built_fiber_tracking_tracking_pft.py>`
```

Comment:
Maybe spell out "Particle Filtering Tractography"?

Reply:
Done in 44c1deb.